src/adapters/kafka/index.ts

```diff
@@ -42,21 +44,15 @@ class KafkaAdapter extends Adapter {
     await producer.connect();
     await consumer.run({
       eachMessage: async ({ topic, partition, message }) => {
-        const gleemessage = new GleeMessage({
-          // topic: topic,
-          // partition: partition,
-          // value: message.value.toString(),
-          // key: message.key,
-          // offset: message.offset,
-          // timestamp: message.timestamp,
-          // headers: message.headers,
-        });
         const prefix = `${topic}[${partition} | ${message.offset}] / ${message.timestamp}`
         console.log(`- ${prefix} ${message.key}#${message.value}`)
```

Comment:
Here you have to create the GleeMessage and pass it over to Glee:

```suggestion
        const gleeMessage = this.createMessage(message)
        this.emit('message', gleeMessage)
```

Then you need to add another function called `createMessage` to this adapter. Similar to the one here: https://github.com/asyncapi/glee/blob/master/src/adapters/mqtt/index.ts#L151. The difference is that instead of getting an MQTT `packet` you get a Kafka `message`. You have to convert the message to a GleeMessage and return it. I think we discussed how to convert to GleeMessage on Slack.